### ISSUE TITLE
Include less files in package

### ISSUE THREAD

**Issue by dougwilson** [blakeembrey/array-flatten#1]
Right now the package contains these files, which I think can be safely excluded:

* `.npmignore`
* `.travis.yml`
* `test.js`

The test file and travis config could at least be added to the `.npmignore` or you can remove the `.npmignore` and use the `files` key in `package.json` for an even slimmer package :)

```js
{
  "files": [
    "array-flatten.js",
    "LICENSE",
    "README.md"
  ]
}
```

(you don't have to add `README.md` explicitly)

**Comment by blakeembrey** [comment]
Thanks for the heads up, I'll need to look at adding this to all my repos :smile:

**Comment by blakeembrey** [comment]
Fixed and release with `v1.0.1`. Thanks!

**Comment by dougwilson** [comment]
Awesome! This is why I'm choosing your flatten module to power Express :)
